addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -260,7 +260,7 @@
             <field name="name">im_livechat.canned_response.tree</field>
             <field name="model">mail.shortcode</field>
             <field name="arch" type="xml">
-                <tree editable="bottom">
+                <tree editable="bottom" default_order="source" sample="1">
```

Comment:
I'm wondering whether this could be removed too and moved to mail views only?

Reply:
I think we can remove those overrides all-together. Done :+1: